mteb/models/gritlm.py

```diff
@@ -14,6 +14,7 @@ def gritlm_loader(**kwargs):
         raise ImportError(
             "GritLM is not installed. Please install it with `pip install gritlm`."
         )
+    kwargs.pop("device", None)  # GritLM does automatic device placement
```

Comment:
What you could do is just remove any kwarg which isn't allowed.

Reply:
That would require a list of kwargs that are allowed right? It's a bit complex cuz the kwargs are passed to HF AutoModel which allows lots of kwargs and may even depend on the model that is loaded I think

Reply:
Ahh right then this fix is fine